dashboard/modules/serve/tests/test_serve_dashboard.py

```diff
@@ -661,6 +662,8 @@ def test_get_applications_while_gcs_down(
 ):
     # Test serve REST API availability when the GCS is down.
     monkeypatch.setenv("RAY_SERVE_KV_TIMEOUT_S", "3")
+    importlib.reload(ray.serve._private.constants)  # to reload the constants set above
```

Comment:
why do we need to change this?

Reply:
By default, Ray Serve uses infinite timeout for internal kv put/get. To test when GCS is down, it sets the timeout to 3s. However this setting never worked because the env did not load.

Previous PythonGcsClient would return error of GrpcUnavailable on GCS down even if timeout is inf. The new GcsClient properly infinitely retries and hangs. To make the env work, we need to reload it.